### PR TITLE
Update Azure AD SSO docs

### PR DESCRIPTION
**Pull request by papunsenapati07**
### What

 user.displayname` or `user.mailnickname` dosen't work on actual scenario, it works well If the user already has a TFE account, however, when a new account needs to be created the claim "username" should be sourced to "user.mail"  or "user.userprincipalname".
### Why

This is important when the user is configuring SSO using azure ad.

Ticket: https://support.hashicorp.com/hc/en-us/requests/80987

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx [hunk 1]
```diff
@@ -88,6 +88,6 @@ If you plan to make use of SAML to set usernames in your Microsoft Azure AD appl
 
 1. Navigate to the single sign-on page.
 1. Edit step 2, "User Attributes & Claims."
```
Comment (mgarrell777) [inline website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx:90]:
```suggestion
1. Edit the **User Attributes & Claims**.
```

Suggest updating for proper user field formatting

File: website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx [hunk 2]
```diff
@@ -88,6 +88,6 @@ If you plan to make use of SAML to set usernames in your Microsoft Azure AD appl
...
-   We recommend naming the claim "Username", leaving the namespace blank, and sourcing something like `user.displayname` or `user.mailnickname`.
+   We recommend naming the claim "Username", leaving the namespace blank, and sourcing something like `user.mail` or `user.userprincipalname`.
 
 If you namespaced any of your claims, note that the attribute name passed by Microsoft Azure AD will follow the form `<claim_namespace/claim_name>`. Consider this when setting Team and Username attribute names.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx:93]:
```suggestion
If you namespaced any of your claims, then Microsoft Azure AD passes the attribute name using the format `<claim_namespace/claim_name>`. Consider this format when setting Team and Username attribute names.
```

Suggest rewording to be more active voice, avoid a future tense style issue, and avoid using "This" without a subject.

**Review by mgarrell777 — APPROVED** [review]
@papunsenapati07 Thanks very much for your effort on this PR. I have a couple small edit suggestions. But overall looks good to me.

File: website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx (hunk 2 above)
Comment (papunsenapati07) [reply website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx]: Thanks mate, I have committed to the suggested changes.

**Review by rkoron007 — APPROVED** [review]
Added the suggestions and updated for the current page.
